babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -191,10 +191,10 @@ class BabelParserTest extends SqlParserTest {
         .ok("SELECT TIMESTAMP '1969-07-20 00:00:00'");
     // PostgreSQL allows the following. We should too.
     sql("select ^timestamp '1969-07-20 1:2'^")
-        .fails("Illegal TIMESTAMP literal '1969-07-20 1:2': not in format "
+        .fails("Illegal TIMESTAMP literal 1969-07-20 1:2: not in format "
```

Comment:
Removing the single-quotes from the error message doesn't seem to be an improvement.

Reply:
Yes, it's worth keeping the error message the same. You probably just need to edit `CalciteResource.java` and `CalciteResource.properties`.

Reply:
Updated based on your suggestion.